shavar-prod-lists/abpindo_adservers.json

```diff
@@ -0,0 +1,993 @@
+{
+    "categories": {
```

Comment:
Extra unused file? Also, if we're using some library to process the json, consider adding them into tools/

Reply:
This is the source of the blocking list json file, gradle build copy this file into app/src/webkit/res/raw/. This is a hard-coded, handcraft, manual parsed JSON from the ABP format rules(https://adblockplus.org/filter-cheatsheet). I think we do need to write a parser if we want to keep the list updated or we need to add extra ABP region.